### PR TITLE
Pass backup flag through Import request options

### PR DESCRIPTION
**Pull request by wayblink**
issue: #20335

Signed-off-by: wayblink <anyang.wang@zilliz.com>

### CODE REVIEW

**Comment by wayblink** [comment]
rerun ut

**Comment by wayblink** [comment]
rerun ut

File: internal/util/importutil/import_options.go [hunk 1]
```diff
@@ -92,3 +93,12 @@ func ParseTSFromOptions(options []*commonpb.KeyValuePair) (uint64, uint64, error
 	}
 	return tsStart, tsEnd, nil
 }
+
+// IsBackup returns if the request is triggered by backup tool
+func IsBackup(options []*commonpb.KeyValuePair) bool {
+	isBackup, err := funcutil.GetAttrByKeyFromRepeatedKV(BackupFlag, options)
+	if err != nil || isBackup != "true" {
```
Comment (congqixia) [inline internal/util/importutil/import_options.go:100]: shall be support case insensitivity here?

Comment (wayblink) [reply internal/util/importutil/import_options.go]: Not necessary, better to have it. Updated

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**